### PR TITLE
add chefSchedule for users and chefs

### PR DESCRIPTION
**Pull request by sarahwithanhwilson**
Also includes some non-working graphQL stuff

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+const bodyParser = require('body-parser');
+// const gqlSchema = require('./schema.js');
 const App = require('../src/app/app').default;
 
 const { StaticRouter } = ReactRouter;
 const baseTemplate = fs.readFileSync(`${__dirname}/../src/index.html`);
 const template = _.template(baseTemplate); // returns a function
 
 const app = express();
-const db = require('./../database/database.js');
```
Comment (dukegoulden) [inline server/server.js]: Do we not need this?

File: src/app/users/UserHome.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
+import ViewChefSchedule from './ViewChefSchedule';
 
-const UserHome = () => <div>User Home View Here</div>;
+const UserHome = () => <div><ViewChefSchedule /></div>;
```
Comment (dukegoulden) [inline src/app/users/UserHome.jsx]: Do you want to put this component somewhere else? Or do you mind if I do when I PR?

File: server/server.js (hunk 1 above)
Comment (sarahwithanhwilson) [reply server/server.js]: I meant to just comment it out, but probably not. With GraphQL we will be querying the database differently. We can add it back later if needed.

File: src/app/users/UserHome.jsx (hunk 2 above)
Comment (sarahwithanhwilson) [reply src/app/users/UserHome.jsx]: Yeah, I just stuck it there to be able to view it and play around with it a bit. You can move it wherever makes sense.

**Pull request merged**